Dockerfile.client

```diff
@@ -0,0 +1,9 @@
+# we don't have a elife base image yet for node, so we use directly the official one
+FROM node:6.13.0-stretch AS client
```

Comment:
not sure yet whether to extract it or not, as we don't have node-based servers, only client stuff